<?php

namespace Container2YcVERS;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getOverblogGraphql_AccessResolverService extends App_KernelDevDebugContainer
{
    /**
     * Gets the public 'overblog_graphql.access_resolver' shared service.
     *
     * @return \Overblog\GraphQLBundle\Resolver\AccessResolver
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/overblog/graphql-bundle/src/Resolver/AccessResolver.php';
        include_once \dirname(__DIR__, 4).'/vendor/webonyx/graphql-php/src/Executor/Promise/PromiseAdapter.php';
        include_once \dirname(__DIR__, 4).'/vendor/webonyx/graphql-php/src/Executor/Promise/Adapter/SyncPromiseAdapter.php';

        return $container->services['overblog_graphql.access_resolver'] = new \Overblog\GraphQLBundle\Resolver\AccessResolver(($container->privates['GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter'] ?? ($container->privates['GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter'] = new \GraphQL\Executor\Promise\Adapter\SyncPromiseAdapter())));
    }
}
